File: Arrays/C/LinearSearch.c
// Linear Search
#include <stdio.h>

int linearSearch(int array[], int size, int target) {
    for (int i = 0; i < size; i++) {
        if (array[i] == target) {
            return i;
        }
    }
    return -1;
}

int main() {
    int array[] = {1, 2, 3, 4, 5};
    int size = sizeof(array) / sizeof(array[0]);
    int target = 3;
    int result = linearSearch(array, size, target);

    if (result != -1) {
        printf("Element found at index %d\n", result);
    } else {
        printf("Element not found in the array\n");
    }

    return 0;
}

File: Arrays/Java/LinearSearch.java
// Linear Search
public class DoubleArray {

    public static void DoubleArray(int[] array) {
        for (int i = 0; i < array.length; i++) {
            array[i] *= 2;
        }

        System.out.println("Array elements after doubling: ");
        for (int i = 0; i < array.length; i++) {
            System.out.print(array[i] + " ");
        }
        System.out.println();
    }

    public static void main(String[] args) {
        int[] array = {1, 2, 3, 4, 5};
        DoubleArray(array);
    }
}